examples/window-app/common/window-app.matter

```diff
@@ -1398,12 +1558,16 @@ endpoint 0 {
 
 endpoint 1 {
   server  cluster Descriptor;
+  server  cluster Groups;
```

Comment:
Please sync with master and regen zap - endpoint info changed for server clusters (we now also show memory mappings)